tests/test_find_iocs.py

```diff
@@ -0,0 +1,51 @@
+from typing import Dict, List
+import pytest
+from ioc_finder import find_iocs
+from .find_iocs_cases import ALL_TESTS
+
+
+@pytest.mark.parametrize("text, results, args", ALL_TESTS)
+def test_find_iocs(text: str, results: Dict, args: Dict):
+    # Parse input
+    iocs = find_iocs(text, **args)
+
+    # Get all keys
+    ioc_finder_keys = _get_nonempty_keys(iocs)
+    # results_keys = _get_nonempty_keys(results)
+    results_keys = list(results.keys())
+
+    # Test output with expected results
+    assert sorted(ioc_finder_keys) == sorted(
+        results_keys
+    ), f"Key values don't match '{ioc_finder_keys} vs '{results_keys}'"
```

Comment:
Minor, add missing quotation:

```suggestion
    ), f"Key values don't match '{ioc_finder_keys}' vs '{results_keys}'"
```